examples/component/http-client/README.md

```diff
@@ -0,0 +1,83 @@
+# TinyGo HTTP Client
+
+This repository contains a WebAssembly Component written in [TinyGo][tinygo], which:
+
+- Implements a [`wasi:http`][wasi-http]-compliant HTTP handler
+- Uses the [`httpserver` provider][httpserver-provider] to serve requests
+- Calls a random number generator service to get random numbers using [`wasi:http`][wasi-http]
+- Can be declaratively provisioned with [`wadm`][wadm]
+
+[wasi-http]: https://github.com/WebAssembly/wasi-http
+[httpserver-provider]: https://github.com/wasmCloud/wasmCloud/tree/main/crates/provider-http-server
+[httpclient-provider]: https://github.com/wasmCloud/wasmCloud/tree/main/crates/provider-http-client
+[wadm]: https://github.com/wasmCloud/wadm
+[tinygo]: https://tinygo.org/getting-started/install/
+[wash]:  https://wasmcloud.com/docs/ecosystem/wash/
+
+# Dependencies
+
+Before starting, ensure that you have the following installed:
+
+- The [TinyGo toolchain][tinygo]
+- [`wash`, the WAsmcloud SHell][wash] installed.
```

Comment:
Also `Go` is needed here.

Reply:
Should we call out `wasm-tools`?

Reply:
Good point, we don't have those in any of them.